build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh

```diff
@@ -74,7 +74,10 @@ export CTEST_PARALLEL_LEVEL=${CTEST_PARALLEL_LEVEL:-$(nproc)}
 tests_passed=true
 
 echo "***** Testing with CTest *****"
-if ! ctest --timeout 900 --output-on-failure \
+if ! ctest \
+   --timeout 900 \
+   --output-on-failure \
+   --no-tests=error \
    --tests-regex "^integrations/tensorflow/|^runtime/bindings/python/" \
    --label-exclude "^nokokoro$|^vulkan_uses_vk_khr_shader_float16_int8$"
```

Comment:
hmmm...

```
[22:20:37 UTC] ctest --timeout 900 --output-on-failure --no-tests=error --tests-regex '^integrations/tensorflow/|^runtime/bindings/python/' --label-exclude '^nokokoro$|^vulkan_uses_vk_khr_shader_float16_int8$'
Test project /home/kbuilder/iree/build/tf
No tests were found!!!
Errors while running CTest
```

Reply:
Looks like `runtime/bindings/python/` should be `iree/runtime/bindings/python` in test names now, though that is already covered by our other CI pipelines. I think we can also drop the `integrations/tensorflow` part entirely, as that is covered by the explicit lit invocation below (no other CMake in that part of the tree now?)

Reply:
I made some tweaks to the `cmake-bazel` pipelines based on what this uncovered. Can you take another look, @GMNGeoffrey ?